packages/truffle-debugger/lib/data/sagas/index.js

```diff
@@ -34,6 +35,7 @@ function *tickSaga() {
   let scopes = yield select(data.info.scopes);
   let definitions = yield select(data.views.scopes.inlined);
   let currentAssignments = yield select(data.proc.assignments);
+  let curDepth = yield select(solidity.current.functionDepth); //for pairing with ids
```

Comment:
Style suggestion: don't abbreviate variable names. Preferably ever, but occasionally it's fine if it's very clear. 

Yep, it's annoying to write; I know—I typed all those long names out everywhere :).

Reply:
Oy, and I was going to change `longId` to `augId` for clarity -- actually I thought I *did* do that but apparently not -- but I guess it should be `augmentedId`? Yuck.  Is `cur`, as in `curDepth`, at least an acceptable abbreviation?  That one should be common enough, right?

Reply:
Maybe I am extreme in my views and just stubbornly refuse to abbreviate anything. I used `currentAssignments`, after all (I did consider `curAssignments`)

Reply:
Anyway, for now I've de-abbreviated all cases of "cur" and "aug".  I don't *think* there were any others I used...